### PR TITLE
Couple bug fixes found when running the solr importer

### PR DESCRIPTION
**Pull request by dannymandel**
These were crashers we hit when running the solr importer

### CODE REVIEW

File: isamples_metadata/SESARTransformer.py [hunk 1]
```diff
@@ -231,7 +231,9 @@ def _primaryLocationType(self) -> typing.AnyStr:
         return None
 
     def _materialType(self) -> typing.AnyStr:
-        return self._source_record_description()["material"]
+        if "material" in self._source_record_description():
```
Comment (datadavev) [inline isamples_metadata/SESARTransformer.py:234]: Can also do something like:
```python
return self._source_record_description().get("material", None)
```
To return a default value if key isn't in dict

File: isamples_metadata/SESARTransformer.py [hunk 2]
```diff
@@ -384,7 +386,10 @@ def sampling_site_elevation(self) -> typing.AnyStr:
         supplement_metadata = self._supplement_metadata()
         if supplement_metadata is not None and "elevation" in supplement_metadata:
             elevation_value = supplement_metadata["elevation"]
-            elevation_unit = supplement_metadata["elevationUnit"]
+            if "elevationUnit" in supplement_metadata:
```
Comment (datadavev) [inline isamples_metadata/SESARTransformer.py:389]: Set above comment on using `dict.get(key, default)`

**Review by datadavev — COMMENTED** [review]
Looks good, just a suggestion that using dict.get(key, default) can be helpful especially where you don't have control over the dict being processed.

File: isamples_metadata/SESARTransformer.py (hunk 1 above)
Comment (dannymandel) [reply isamples_metadata/SESARTransformer.py:234]: Oh nice, thanks for the suggestion.  Much more concise!

**Pull request merged**